exercises/bob/example.rs

```diff
@@ -6,13 +6,14 @@ pub fn reply(message: &str) -> &str {
 }
 
 fn is_silence(message: &str) -> bool {
-    message == ""
+    message.trim().is_empty()
 }
 
 fn is_yelling(message: &str) -> bool {
-    message.chars().all(|char| !char.is_lowercase())
```

Comment:
don't really care since the PR as written passes the tests, but that means it was possible to keep `!is_lowercase`:


```rust
message.chars().any(char::is_alphabetic) && message.chars().all(|c| !c.is_lowercase())
```

Or its equivalent

```rust
message.chars().any(char::is_alphabetic) && !message.chars().any(char::is_lowercase)
```